### PR TITLE
Update unattended upgrades role

### PR DESCRIPTION
**Pull request by tersmitten**
Requires a recent (the last) version of Frey

### CODE REVIEW

File: .infra/Freyfile.hcl [hunk 1]
```diff
@@ -118,7 +118,7 @@ install {
     }
 
     roles {
-      role = "{{{init.paths.roles_dir}}}/unattended-upgrades/v1.2.0"
+      role = "{{{init.paths.roles_dir}}}/unattended-upgrades/1.3.0"
```
Comment (tersmitten) [inline .infra/Freyfile.hcl]: Only the line above is relevant

**Comment by tersmitten** [comment]
This is based of `mts-webdev:make-use-of-forked-roles`

**Comment by kvz** [comment]
Seems there is a conflict here

**Comment by tersmitten** [comment]
I'll fix it

**Comment by tersmitten** [comment]
Done

**Pull request merged**